File: pingyougou_parent/pingyougou_manager_Web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    //读取列表数据绑定到表单中
    this.findAll = function () {
        return $http.get("../brand/findAll.do");
    }
    this.findPage = function (pageNum,pageSize) {
        return   $http.get('../brand/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
    }
    this.add = function (entity) {
        return   $http.post('../brand/add.do',entity)
    }
    this.update = function (entity) {
        return   $http.post('../brand/update.do',entity)
    }
    this.update1 = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    this.dele = function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds)
    }
    this.search = function (pageNum,pageSize,searchEntity) {
        return $http.post('../brand/findSearch.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity)
    }

})

File: pingyougou_parent/pingyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

/*
*   品牌接口
* */
public interface  BrandService {
    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum, int pageSize);

    public  void add(TbBrand tbBrand);

//    根据id 查询数据
    public  TbBrand findOne(Long id);
//     修改数据
    public  void update(TbBrand tbBrand);
//    删除数据
    public void delete(Long[] ids);
//   分页带查询
public PageResult findSearch(TbBrand tbBrand, int pageNum, int pageSize);
}
